src/ai-bundle/src/AiBundle.php

```diff
@@ -395,6 +396,25 @@ private function processPlatformConfig(string $type, array $platform, ContainerB
             return;
         }
 
+        if ('albert' === $type) {
+            $platformId = 'ai.platform.albert';
+            $definition = (new Definition(Platform::class))
+                ->setFactory(AlbertPlatformFactory::class.'::create')
+                ->setLazy(true)
+                ->addTag('proxy', ['interface' => PlatformInterface::class])
+                ->setArguments([
+                    $platform['api_key'],
+                    $platform['base_url'],
+                    new Reference($platform['http_client'], ContainerInterface::NULL_ON_INVALID_REFERENCE),
+                    new Reference('event_dispatcher'),
```

Comment:
No need to set a ModelCatalog?

Reply:
I updated the factory so it now accepts a ModelCatalog. This makes the catalog configurable when needed.

I also saw that the ModelCatalog alignment for Albert has already been merged (#839), so that part is now resolved. Nothing to handle here on my side anymore.